tests/test_spawn.py

```diff
@@ -111,3 +113,24 @@ def test_interactive_repl_unicode_noecho(self):
     @unittest.skipIf(which('bc') is None, "bc(1) not found on this server.")
     def test_interactive_repl_unicode_echo(self):
         self._interactive_repl_unicode(echo=True)
+
+    def test_pass_fds(self):
+        with tempfile.NamedTemporaryFile() as temp_file:
+            temp_file_fd = temp_file.fileno()
+            temp_file_name = temp_file.name
+
+            # Temporary files are CLOEXEC by default
+            fcntl.fcntl(temp_file_fd,
+                        fcntl.F_SETFD,
+                        fcntl.fcntl(temp_file_fd, fcntl.F_GETFD) &
+                        ~fcntl.FD_CLOEXEC)
+
+            p = PtyProcess.spawn(['bash',
+                                  '-c',
+                                  'printf hello >&{}'.format(temp_file_fd)],
+                                 echo=True,
+                                 pass_fds=(temp_file_fd,))
```

Comment:
Is it worth having a test that when you don't specify it in `pass_fds`, the child process can't access it?

Reply:
I think that the more tests, the better :) I've added that one too.